lib/bloom_remit_client/responses/get_deposit_targets_response.rb

```diff
@@ -0,0 +1,12 @@
+module BloomRemitClient
+  class GetDepositTargetsResponse < BaseResponse
+
+    def default_parsed_body
```

Comment:
When I read `parsed_body` I expect a hash to be returned, not deposit target models. You can make a new method `#deposit_targets` that loops through parsed body, like what this one does